xbmc/utils/StringUtils.cpp

```diff
@@ -1297,7 +1297,7 @@ std::string StringUtils::FormatFileSize(uint64_t bytes)
 {
   const std::array<std::string, 6> units{"B", "kB", "MB", "GB", "TB", "PB"};
   if (bytes < 1000)
-    return Format("%lluB", bytes);
+    return Format("%"PRIu64"B", bytes);
```

Comment:
well, we have this check in other places already and android is always a candidate for toolchain weirdness.